packages/workspace/src/browser/workspace-uri-contribution.ts

```diff
@@ -99,10 +102,6 @@ export class WorkspaceUriLabelProviderContribution extends DefaultUriLabelProvid
             return super.getLongName(uri);
         }
 
-        const short = uriStr.substr(this.wsRoot.length);
-        if (short[0] === '/') {
-            return short.substr(1);
-        }
-        return short;
+        return uriStr.substr(this.wsRoot.length);
```

Comment:
How about getting a relative path: https://github.com/theia-ide/theia/blob/41b4d633ac04668a9e8c7068373a1e6494a5a66f/packages/core/src/common/uri.ts#L65

It is already well tested.

Reply:
Makes sense.  I updated the PR.